zcash_client_sqlite/src/wallet/init/migrations.rs

```diff
@@ -267,9 +269,19 @@ pub const V_0_15_0: &[Uuid] = &[
     v_transactions_additional_totals::MIGRATION_ID,
 ];
 
-const V_0_16_0: &[Uuid] = &[transparent_gap_limit_handling::MIGRATION_ID];
+/// Leaf migrations in the 0.16.0 release.
+const V_0_16_0: &[Uuid] = &[
+    fix_bad_change_flagging::MIGRATION_ID,
+    v_transactions_additional_totals::MIGRATION_ID,
+    transparent_gap_limit_handling::MIGRATION_ID,
+];
 
-const V_0_16_2: &[Uuid] = &[ensure_default_transparent_address::MIGRATION_ID];
+/// Leaf migrations in the 0.16.2 release.
+const V_0_16_2: &[Uuid] = &[
+    fix_bad_change_flagging::MIGRATION_ID,
+    v_transactions_additional_totals::MIGRATION_ID,
+    ensure_default_transparent_address::MIGRATION_ID,
+];
```

Comment:
These changes fix a bug where the text graph correctly showed there was more than one leaf migration (though incorrectly only two), but the added constants only declared one of them.

Reply:
Okay, I wasn't sure what exactly to do here given that the same migration was updated in multiple releases.